apps/admin-web/src/components/pages/tools/Events/EventForm.tsx

```diff
@@ -1,9 +1,12 @@
 import React, { useGlobal, useState } from "reactn";
 
-import { useMutation } from "@apollo/react-hooks";
 import moment from "moment";
 
-import { CREATE_EVENT, GET_EVENTS, UPDATE_EVENT } from "./helpers";
+import { GET_EVENTS } from "./helpers";
```

Comment:
Do you still need this import with the generated stuff?

Reply:
As above